### PR TITLE
Major version bump.

### PR DESCRIPTION
**Pull request by somexpert**
Brings the version to 2.0.0 due to the app no longer being compatible with Django 1.5-1.7. README, docs, and setup have been updated accordingly.

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, @runderwood

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Invite is a Django app for inviting new users to your new or existing Django pro
 ## Dependencies
 
 * Python 2.7.x
-* Django 1.5 - 1.7
+* Django 1.8 (Please use django-invite 1.0.2 for Django 1.5-1.7)
 
 ## Documentation
```
Comment (ldko) [inline README.md]: We could add in steps for 
`./manage.py migrate`
`./manage.py createsuperuser`
before the  "Run the development server." step.

Comment (somexpert) [inline README.md]: :boom:

File: docs/conf.py [hunk 2]
```diff
@@ -51,9 +51,9 @@
 # built documents.
```
Comment (ldko) [inline docs/conf.py]: We could change the copyright date in here and LICENSE to 2017.

**Comment by ldko** [comment]
While we are changing this stuff, I think there is a typo in "Where your users ld be redirected to upon logging out from within the Invite app." of  [configuration.rst](https://github.com/unt-libraries/django-invite/blob/master/docs/configuration.rst).

File: docs/quickstart.rst [hunk 3]
```diff
@@ -12,10 +12,10 @@ Requirements
 Installation
```
Comment (ldko) [inline docs/quickstart.rst]: "install" should be "installed" in line 11.

**Pull request merged**